File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function($http){
	//全部查询
	this.findAll=function(){
	return $http.get("../brand/findAll.do");
	}
	//分页
	this.findPage=function(page, rows){
		return $http.get('../brand/findPage.do?page=' + page + '&rows=' + rows);
	}
	//保存
	this.add=function(entity){
		return $http.post('../brand/add.do', entity);
	}
	//修改
	this.update=function(entity){
		return $http.post('../brand/update.do', entity);
	}
	//根据id查询
	this.findOne=function(id){
		return $http.get('../brand/findOne.do?id=' + id);
	}
	//批量删除
	this.deleteBrands=function(ids){
		return $.get('../brand/delete.do?ids=' + ids);
	}
	//条件查询
	this.search=function(page,rows,searchEntity){
		return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
	}
	//下拉列表
	this.findBrandList=function(){
		return $http.get('../brand/findBrandList.do');
	}
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

/**
 * 品牌的接口
 * 
 * @author Administrator
 *
 */
public interface BrandService {
	/**
	 * 返回所有品牌
	 * 
	 * @return
	 */
	public List<TbBrand> findAll();

	/**
	 * 分页列表
	 */
	public PageResult findPage(int pageNum, int pageSize);

	/**
	 * 添加功能
	 */
	public void addBrand(TbBrand brand);

	/**
	 * 修改功能.先查出来，在修改
	 */
	public TbBrand findOne(long id);

	public void update(TbBrand brand);

	/**
	 * 批量删除
	 */
	public void delete(long[] ids);
	/**
	 * 条件查询
	 */
	public PageResult findPage(TbBrand brand,int pageNum, int pageSize);
		
	/**
	 * 下拉列表
	 */
	public List<Map> findBrandList();

}
